links.md

```diff
@@ -8,54 +8,25 @@ permalink: /links/
 
 
 ### RSE Associations 
-  - <a class="" target="_blank" 
-  			href="https://society-rse.org/">
-  			Society of Research Software Engineering
-  			</a>
-  - <a class="" target="_blank" 
-  			href="https://de-rse.org/">
-  			Germany RSE Association
-  			</a>
-  - <a class="" target="_blank" 
-  			href="https://nl-rse.org/">
-  			Netherlands RSE Association
-  			</a>
-  - <a class="" target="_blank" 
-  			href="https://nordic-rse.org/">
-  			Nordic RSE Association
-  			</a>
-  - <a class="" target="_blank" 
-  			href="https://us-rse.org/">
-  			USA RSE Association
-  			</a>
-
-  
+  - <a class="" target="_blank" href="https://society-rse.org/">Society of Research Software Engineering</a>
+  - <a class="" target="_blank" href="https://de-rse.org/">Germany RSE Association</a>
+  - <a class="" target="_blank" href="https://nl-rse.org/">Netherlands RSE Association</a>
+  - <a class="" target="_blank" href="https://nordic-rse.org/">Nordic RSE Association</a>
+  - <a class="" target="_blank" href="https://us-rse.org/">USA RSE Association</a>
+
+
 ### RSE Meetings
```

Comment:
May be we should rename to "Past RSE Meetings"

Reply:
Done.